### ISSUE TITLE
Security against meet-in-the-middle attacks

### ISSUE THREAD

**Issue by CodesInChaos** [veorq/ipcrypt#2]
Doesn't this cipher suffer from the same meet-in-the-middle attack as 2DES with complexity 2^64?  (though the memory requirements might make implementing that attack not as easy as it sounds)

**Comment by veorq** [comment]
Yeah you can do this, but a codebook attack in 2^32 is even more efficient :)
